scripts/ci/openshift-deploy.sh

```diff
@@ -126,6 +131,18 @@ git clone https://github.com/operator-framework/operator-test-playbooks.git
 cd operator-test-playbooks/upstream
 export ANSIBLE_CONFIG=/tmp/playbooks2/operator-test-playbooks/upstream/ansible.cfg
 ANSIBLE_STDOUT_CALLBACK=yaml ansible-playbook -i localhost, deploy-olm-operator-openshift-upstream.yml -e ansible_connection=local -e package_name=$OP_NAME -e operator_dir=$TARGET_PATH/$OP_NAME -e op_version=$OP_VER -e oc_bin_path="/tmp/oc-$OC_DIR_CORE/bin/oc" -e commit_tag=$QUAY_HASH -e dir_suffix_part=$OC_DIR_CORE $SUBDIR_ARG $EXTRA_ARGS -vv
+if [ $? -eq 0 ]; then
+  curl -f -u framework-automation:$(cat /var/run/cred/framautom) \
+  -X POST \
+  -H "Accept: application/vnd.github.v3+json" \
+  https://api.github.com/repos/operator-framework/community-operators/dispatches --data "{\"event_type\": \"openshift-test-status\", \"client_payload\": {\"source_pr\": \"$PULL_NUMBER\", \"remove_labels\": [\"openshift-started\", \"deployment-ok\"], \"add_labels\": [\"deployment-ok\"]}}"
+else
+  curl -f -u framework-automation:$(cat /var/run/cred/framautom) \
+  -X POST \
+  -H "Accept: application/vnd.github.v3+json" \
+  https://api.github.com/repos/operator-framework/community-operators/dispatches --data "{\"event_type\": \"openshift-test-status\", \"client_payload\": {\"source_pr\": \"$PULL_NUMBER\", \"remove_labels\": [\"openshift-started\", \"deployment-ok\"], \"add_labels\": [\"openshift-failed\"]}}"
+fi
```

Comment:
There is lot of code duplication in if else block.
It is difficult to spot what is a difference. It look like the only difference is `\"add_labels\": [\"deployment-ok\"]` vs `\"add_labels\": [\"openshift-failed\"]`

Please reduce code duplication.

it can be something like 
```
if [ $? -eq 0 ]; then
    LABLES_PART = # ok
else 
   LABLES_PART = # bad
fi

curl ... --data "{  //snip common par ot of json
                       \"add_labels\": $LABLES_PART}"
```